docs/install/install-local-migrate.md

```diff
@@ -87,89 +86,65 @@ Prior to attempting this step, do consider the following:
+*  If you have larger Fedora collections, there is a possibility that you made changes to the `akubra-llstore.xml` file to allow for the creation of larger or deeper hash directories)  
+
+  * You will need to copy your `/usr/local/fedora/server/config/spring/akubra-llstore.xml` from your Production Fedora System to `./config/fedora/akubra-llstore.xml`  
+
   * You will then need to add an extra line in the Fedora service (fedora) volumes section to bind mount this file in. This will guarantee proper Fedora data hash structure.
     * `- ./config/fedora/akubra-llstore.xml:/usr/local/fedora/server/config/spring/akubra-llstore.xml`
 
-#### Solr schema & Islandora transforms
+#### Solr Schema & Islandora Transforms
 
 This data can be challenging depending on the level of customizations to contend with and as such, ISLE maintainers recommends following one of the three strategies outlined below.
 
-##### (**Easy**) - Run "stock" ISLE
+##### **Easy** - Run "Stock" ISLE
 
-Don't copy any existing production Solr schemas, Gsearch xslts etc and opt instead to use ISLE's default versions. Import some objects from your existing Fedora repository and see if they display properly in searches as you like.
+Don't copy any existing production Solr schemas, GSearch .xslt files, etc., and opt instead to use ISLE's default versions. Import some objects from your existing Fedora repository and see if they display properly in searches as you like.
 
-##### (**Intermediate**) - Bind mount in existing transforms and schemas
+##### **Intermediate** - Bind Mount in Existing Transforms and Schemas
 
 Bind mount in existing transforms and schemas  to override ISLE settings with your current Production version.
 
-**WARNING** _Assumes you are running Solr 4.10.x., **only attempt** if you are running that version on Production._
+**WARNING** _This approach assumes you are running Solr 4.10.x.; **only attempt** if you are running that version on Production._
 
 * Copy these current production files and directory to your local laptop in an appropriate location.
   * Solr `schema.xml`
-  * Gsearch `foxmltoSolr.xslt` file
-  * Gsearch `islandora_transforms`
+  * GSearch `foxmltoSolr.xslt` file
+  * GSearch `islandora_transforms`
 
-* Once Step 2 has been completed and the ISLE project is cloned to your laptop and checked into git, create a new directory in:
-  * `config/solr`
-  * `config/fedora/gsearch`
+  * Keep the files you create during this process; you will need them again for `Step 2a` (below)!
 
-* Move your `schema.xml` file into `config/solr/`
+* **Please note:** You may need to further review paths in the files mentioned above, and edit them to match ISLE system paths.  
 
-* Move your `foxmltoSolr.xslt` file & `islandora_transforms` directory into `config/fedora/gsearch/`
+##### **Advanced** - Diff & Merge Current Production Customization Edits into ISLE Configs
+* Copy these current production files and directory to your local laptop in an appropriate location.
```

Comment:
`laptop` should perhaps just be `computer`

Reply:
Agreed, and we probably should not be using ampersands (&) in the docs in this fashion either.

So, should I make these changes locally, commit, push to my fork, and then...?